File: C#/1008.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int num = int.Parse(Console.ReadLine());
        int horas = int.Parse(Console.ReadLine());
        double valor = double.Parse(Console.ReadLine());

        // $ é o formatador de string. o metodo tostring faz ele ficar com x casas decimais
        Console.WriteLine($"NUMBER = {num}");
        Console.WriteLine($"SALARY = U$ {(valor*horas).ToString("0.00")}");
    }

}

File: JS/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function main(){
    const num = parseInt(lines.shift());
    const horas = parseInt(lines.shift());
    const valor = parseFloat(lines.shift());
    
    console.log(`NUMBER = ${num}`);
    console.log(`SALARY = U$ ${(horas*valor).toFixed(2)}`);
}

main();